Toolset/libraries/revidelibrary.8.livecodescript

```diff
@@ -1383,10 +1383,42 @@ end revIDEGlobalVariables
 
 function revIDEGlobalProperties
    set the itemdel to slash
-   set the defaultfolder to item 1 to -3 of the filename of me
+   set the defaultfolder to item 1 to -4 of the filename of me
    return URL("file:globalProperties.txt")
```

Comment:
Is there a revIDESpecialFolderPath we can use or implement instead of item 1 to -4 of the filename of me?